modules/rostests/apitests/gdi32/CreateBitmap.c

```diff
@@ -189,8 +162,8 @@ void Test_CreateBitmap()
         ok(hbmp != 0, "should get a 1x2 bitmap %ld\n", cBitsPixel);
         ok_int(GetObject(hbmp, sizeof(bitmap), &bitmap), sizeof(BITMAP));
 
-        /* calculate expected line width */
-        cjWidthBytes = ((((ULONG)bitmap.bmWidth) * ((ULONG)bitmap.bmBitsPixel) + 15) & ~15) >> 3;
+        /* Calculate expected line width */
+        cjWidthBytes = ((bitmap.bmWidth * bitmap.bmBitsPixel + 15) & ~15) >> 3;
```

Comment:
I can't tell if this is cosmetic or to fix a test failure.

Reply:
Cosmetic.